commands/lint.ts

```diff
@@ -73,7 +82,13 @@ export default class LintCommand extends BaseCommand {
 
   async lintRustFiles() {
     const dockerShellCommandExecutor = await this.dockerShellCommandExecutor("rust-tools");
-    await dockerShellCommandExecutor.exec(`find . -name '*.rs' | xargs rustfmt --edition "2021" --check`);
+    await dockerShellCommandExecutor.exec(`find . -name '*.rs' -exec rustfmt --edition "2021" --check -- {} +`);
+  }
+
+  async lintDockerFiles() {
+    const dockerShellCommandExecutor = await this.dockerShellCommandExecutor("docker-tools");
+    await dockerShellCommandExecutor.exec(`sed -i "/^COPY /s/--exclude=[^ ]*//g" dockerfiles/*.Dockerfile`);
```

Comment:
@ryan-gang will this mutate files in the directory? If so we should probably copy it elsewhere?